poetry/core/masonry/api.py

```diff
@@ -31,7 +31,7 @@ def get_requires_for_build_wheel(config_settings=None):
 
 
 def prepare_metadata_for_build_wheel(metadata_directory, config_settings=None):
-    poetry = Factory().create_poetry(Path(".").resolve())
+    poetry = Factory().create_poetry(Path(".").resolve(), with_dev=False)
```

Comment:
I do not think these tests need to change.

Reply:
Because this isn't test code I'm a little confused what you mean. Could you elaborate?

Reply:
Ignore me.